draft-ietf-rats-corim.md

```diff
@@ -2089,112 +2063,139 @@ Logically, new ECT entries are appended to the existing ACS.
+
+When comparing two `$crypto-key-type-choice` fields for equality, the verifier SHALL treat then as equal if their CBOR encoding is binary equal.
+
+The verifier MAY treat two keys as equal if they have different formats but represent the same key.
+For example, the verifier may contain code to compare a key fingerprint against the key which, when hashed, creates that fingerprint.
+
 ### Claims Comparison  {#sec-compare-claims}
 
-#### Comparison of measurement-values-map {#sec-match-one-codepoint}
+The verifier SHALL iterate over all entries in the candidate entries array and compare the condition ECT claims to the candidate entry authority claims.
+
+The Verifier SHALL iterate over the codepoints which are present in the `measurement-values-map` field within the condition ECT `measurement-map`.
+Each of the codepoints present in the condition ECT is compared against the same codepoint in the candidate entry.
 
-This section describes the algorithm used to compare the `measurement-values-map` codepoints of an ECT with another ECT.
-The comparison algorithm performed depends on the value of the codepoint being compared.
+If any codepoint present in the condition ECT `measurement-values-map` does not have a corresponding codepoint within the candidate entry `measurement-values-map` then verifier SHALL remove that candidate entry from the candidate entries array.
 
-[^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/203
+If any codepoint present in the condition ECT `measurement-values-map` does not match the same codepoint within the candidate entry `measurement-values-map` then verifier SHALL remove that candidate entry from the candidate entries array.
 
-If the `measurement-values-map` value has an associated CBOR tag, the comparison algorithm should comprehend the structure identified by the CBOR tag.
+#### Comparison of a single measurement-values-map codepoint{#sec-match-one-codepoint}
 
-If the Verifier does not recognize a CBOR tag value then the value MUST NOT match.
+If the codepoint key is zero or positive then the Verifier SHALL use a standard comparison algorithm, defined in this document, to compare the claims.
+The codepoint number and, if present, the tag value associated with the condition ECT codepoint SHALL be used to select the algorithm.
 
-Note: CBOR tags are useful for discriminating values amongst alternates, but the comparison of tagged values is still determined by the codepoint they appear under. It is recommended but not required to compare values with a specific CBOR tag the same way across codepoints. For this reason, it is recommended to specify a default comparison algorithm with the CBOR tag's registration.
+If the codepoint key is negative then the Verifier MAY use a comparison algorithm defined by the appropriate CoRIM profile to compare the claims.
+The codepoint number, the profile associated with the condition ECT, and the tag value (if present) shall be used to select the algorithm.
```

Comment:
```suggestion
The codepoint number and the profile associated with the condition ECT may be used to select the algorithm.
```
Again the algorithm is specific to the codepoint and may choose to further dispatch based on value tags.